config/cable.yml

```diff
@@ -0,0 +1,18 @@
+# development:
+#   adapter: async
+#
+# test:
+#   adapter: async
+#
+# production:
+#   adapter: redis
+#   ur: redis-url
+#   channel_prefix: manager_production
+
+redis: &redis
+  adapter: redis
+  url: redis://localhost:6379/3
```

Comment:
Does this need to be configurable for the cloud?

Reply:
Yeah I'll make it an environment variable.